File: C/006-sum-square-difference.c
#include <stdio.h>
#include <math.h>

int main()
{
    int n=100, sumSqr=0, sqrSum = 0, sum=0;
    if(n<=0){
    printf("Invalid value of n");
    }
       else{
          // Use pow to raise the initial value
            for(int i = 1; i <= n; i++) {
                //The sum of the squares 
                sumSqr = sumSqr + pow(i,2);
                sum=sum+i; 
            }
            //The square of the sum 
            sqrSum = pow(sum,2);
            int result = sqrSum-sumSqr;

	    printf("Result = %d\n", result);
       }

    return 0;
}

File: Python/006-sum-square-difference.py
'''
	The sum of the squares of the first ten natural numbers is,
	1² + 2² + ... + 10² = 385

	The square of the sum of the first ten natural numbers is,

	(1 + 2 + ... + 10)² = 552 = 3025
	Hence the difference between the sum of the squares of the first ten natural numbers and the square of the sum is 3025 − 385 = 2640.

	Find the difference between the sum of the squares of the first n natural numbers and the square of the sum.
'''

def sumSquareDifference(number):
	sumOfSquares = 0
	sums = 0

	for i in range(1, number + 1):

		sumOfSquares += i ** 2
		sums += i
	
	squareOfSums = sums ** 2

	return squareOfSums - sumOfSquares


result = sumSquareDifference(10)
print(result)